cli_main.py

```diff
@@ -347,9 +344,9 @@ def main(
 
     # Convert special types (i.e AtomGroups)
     # Ugly that we have to parse again... but currently I have no better idea :(
-    for doc_param in NumpyDocString(analysis_callable.__doc__)["Parameters"]:
-        if "AtomGroup" in doc_param.type:
-            analysis_kwargs[doc_param.name] = u.select_atoms(analysis_kwargs[doc_param.name])
+    for doc_param in parse_docs(analysis_callable)[2]:
```

Comment:
I was reviewing this code today. I have to admit I am a bit lost here with what we have done :stuck_out_tongue: 

A couple of questions (sorry, I blanked):

* what exactly is the purpose of this part?
* Is there a key named `"name"` ? I could not find it.

Don't worry if we need to reparse it. We can think about it later. Maybe we can do this job during the `add_to_CLI` part.